proactive/ProactiveGateway.py

```diff
@@ -162,8 +162,21 @@ def getJobResult(self, job_id, timeout=60000):
         return '\n'.join(v for v in all_results)
 
     def getTaskResult(self, job_id, task_name, timeout=60000):
+        task_result = self.proactive_scheduler_client.waitForTask(str(job_id), task_name, timeout)
+        if type(task_result.getValue()) is bytes:
+            task_result = task_result.getValue().decode('ascii')
```

Comment:
Are you sure it should be ascii? Maybe utf-8?

Reply:
I agree with you. I just wanted to be the closest possible to what Andrews and Marco used before in the `getJobResult()` method. I will follow Mael's suggestion.